scripts/ska3_builder

```diff
@@ -10,20 +10,20 @@ if os.uname().sysname == "Darwin":
 
 parser = argparse.ArgumentParser(description="Build Ska Conda packages.")
 
-parser.add_argument("mode", type=str, help="The build mode. Either 'all' for all "
-                                           "packages, 'updated' for updated "
-                                           "packages, or a specific package name.")
-parser.add_argument("--build_root", default=".", type=str, 
+parser.add_argument("package", type=str, nargs="?",
+                    help="Package to build.  All we be built as needed if no package supplied")
```

Comment:
`All will be built if no package supplied.`  ("As needed" is not needed).

Reply:
That's not strictly true if we use the "--skip-existing" flag to conda build (which I like).  I think it will only build if there is a new tag or if the recipe has changed, but  haven't explicitly checked the "recipe has changed" path.

Reply:
Sorry.  `All updated packages will be ...`.  The recipe changing is a corner case that doesn't need to be explicitly mentioned or tested now.

Reply:
OK!  "All updated packages will be built" sounds a bit circular to me (this is called the package update process in the docs,  but I'll run with that.